debian/changelog

```diff
@@ -1,3 +1,10 @@
+cloud-init (24.1.3-0ubuntu4) noble; urgency=medium
```

Comment:
Since this is our first SRU to noble, I think this need to be changed to `24.1.3-0ubuntu3~24.04.1`.

Reply:
Since we're in the stange space where we are in 0-day SRU territory and devel release hasn't opened yet. Let's go for ubuntu3.1.  If we get suggestions otherwise on upload we'll adjust for the ` ~24.04.1` suffix.